tests/unit/components/EmailTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace prime\tests\components;
+
+use Yii;
+use yii\symfonymailer\Message;
+
+final class EmailTest extends \Codeception\Test\Unit
+{
+    protected function _before()
+    {
+        //\Yii::$app->mailer->messageClass = \yii\symfonymailer\Message::class;
+    }
+
+    protected function _after()
+    {
+    }
+
```

Comment:
Why are there empty functions, without return types? -> I think this is the 10th time I've mentioned return types on every function?